FIP-0032: Make the verify_consensus_fault cost constant (#333)
filecoin-project/FIPs: FIPS/fip-0032.md

### Extern-traversing syscall fee revision

Some syscalls cannot be resolved entirely within FVM space. Such syscalls

> need to traverse the "extern" (FVM<>node) boundary to access client

Reviewer: I think we use `FVM<>client` elsewhere in this doc. Should be consistent. I don't have strong opinions on which to settle on.

Author: Lol we say `FVM<>client` twice and `FVM<>node` thrice, so I thought this was a step towards consistency.

- `get_chain_randomness`: `<<TODO>>`
- `get_beacon_randomness`: `<<TODO>>`

> - `verify_consensus_fault`: `<<TODO>>`. This is a fixed cost that covers all blockstore operations,
> signature verifications, and other logic needed to resolve whether a consensus fault has occured.
> Note that this is different from the current (network v15 and earlier)
> in which each blockstore operation is charged.

Reviewer: ```suggestion
- `verify_consensus_fault`: `<<TODO>>`. This is a fixed cost that covers all blockstore operations,
  signature verifications, and other logic needed to resolve whether a consensus fault has occured.
  Note that differs from network v15 and earlier, where the cost is variable depending on the exact
  blockstore operations performed.
```
